### DISCUSSION TITLE
jinja2 Compatibility with Python 3.11.x

### DISCUSSION THREAD

**Discussion by kmaheshwari-splunk** [pallets/jinja#1832] [Q&A] [answered]
Hello folks,
The [Pypi documentation](https://pypi.org/project/Jinja2/) mentions that jinja2 v3.1.2 is compatible with Python >= 3.7.
Does this claim that the library is compatible with Python 3.11.x as well? 
If not, is there a plan to release a Python 3.11.x compatible version?

Thanks!

**Comment by ThiefMaster** [comment, marked as answer]
Do you have any reason to believe it's not compatible? :)

Our CI runs the tests on 3.11 and even 3.12-dev.

**Reply by kmaheshwari-splunk** [reply]
Hi @ThiefMaster ,
Thanks for quick response!

We just wanted to confirm that the library is compatible with 3.11. No other reasons as such.
